pkgs/development/python-modules/requests-gssapi/default.nix

```
@@ -0,0 +1,27 @@
+{ fetchPypi
+, lib
+, python3Packages
+}:
+python3Packages.buildPythonPackage rec {
```

```suggestion
}:

buildPythonPackage rec {
```

and add `buildPythonPackage` as input.